src/depth-to-rgb-calibration.cpp

```diff
@@ -184,8 +184,12 @@ rs2_calibration_status depth_to_rgb_calibration::optimize(
 
         AC_LOG( DEBUG, "Optimization successful!" );
         _thermal_intr = _algo.get_calibration().get_intrinsics();
-        _thermal_intr.model = RS2_DISTORTION_INVERSE_BROWN_CONRADY;  // restore LRS model
 
+#ifdef __SSSE3__
+        _thermal_intr.model = RS2_DISTORTION_INVERSE_BROWN_CONRADY;  // restore LRS model
+#else
+        _thermal_intr.model = RS2_DISTORTION_BROWN_CONRADY;  // restore LRS model
```

Comment:
Not needed -- just don't change it to begin with

Reply:
done